recipes/seqan3/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "SeqAn3" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/seqan/{{ name|lower }}/releases/download/{{ version }}/seqan-{{ version }}-with-submodules.tar.gz
+  sha256: ba8f0fd5d19289f95823235b69fdc5a12aba3578292463eecf750722e8662bd3
+  patches:
+    - bioconda.patch
+
+build:
+  noarch: generic
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('seqan3', max_pin='x.x') }}
```

Comment:
This is a header-only library now, yes? I don't think we need a run-export then. Will this require anything else during runtime though? That is, does using seqan require anything like zlib, or would that be from the program that uses seqan then?

@bgruening Shouldn't the run-export set the run-time pinning, which won't actually be relevant?

Reply:
Yes, run-exports will set run-time pinning. If this is a header-only package it is only used during build-time isn't it?

Reply:
This is a header only library, and we will use zlib/bzip2 if it is available (I'd actually prefer those packages to be installed together with this package, since stuff like BAM support relies on this).

Reply:
Should we add zlib/bzip2 to the run time and then a run_exports for them then? I'm not entirely sure how all of that works... :(

Reply:
They will only end up in the run-time deps but we will need them in the host section. I don't think this helps. We should remove the run_exports here.

Reply:
I guess if user-programs end up needing zlib/bzip2 they'll notice that anyway during testing, add that to the host/run and then the normal pinning work resolve the issues. So I guess we don't need to worry about somehow shoehorning them in here.

Reply:
Like this, or both host and run requirements?